types/context_test.go

```diff
@@ -97,7 +97,7 @@ func (s *contextTestSuite) TestContextWithCustom() {
 	headerHash := []byte("headerHash")
 	zeroGasCfg := storetypes.GasConfig{}
 
-	ctx = types.NewContext(nil, header, ischeck, logger)
+	ctx = types.NewContextWithHeader(nil, header, ischeck, logger)
```

Comment:
I think the header we pass here is empty, so we could just use the other constructor